### PR TITLE
Add launch argument to avoid pointcloud padding

### PR DESCRIPTION
**Pull request by ivanpauno**
Connected to https://github.com/ros-perception/image_pipeline/pull/524.

### CODE REVIEW

File: launch/bringup.launch.xml [hunk 1]
```diff
@@ -4,6 +4,7 @@
   <arg name="start_gazebo" default="false" />
   <arg name="viz" default="false" />
   <arg name="stereo_pipeline_single_process" default="true" />
+  <arg name="avoid_point_cloud_padding" default="True" />
```
Comment (ivanpauno) [inline launch/bringup.launch.xml]: Maybe, the default should still be `False`.

**Comment by ivanpauno** [comment]
@jacobperron Do you think this is ready to be merged?

File: launch/bringup.launch.xml (hunk 1 above)
Comment (jacobperron) [reply launch/bringup.launch.xml]: I think defaulting to true is fine; this is just a playground repository.

nitpick: lowercase "true" to match the other arguments.

Comment (jacobperron) [inline launch/bringup.launch.xml]:
```suggestion
  <arg name="avoid_point_cloud_padding" default="true" />
```

**Pull request merged**